cmd/main_test.go

```diff
@@ -122,8 +122,9 @@ var staticTests = []*cliTest{
 		expectedOutFile: []string{"examples/output/topology.txt", "examples/output/analysis.txt"},
 	},
 	{
-		name:            "synthesize-only",
-		args:            "--resource-input-file ../pkg/collector/data/json/Example1.json --synthesis-dump-dir examples/output/synthesis",
+		name: "synthesize-only",
+		args: "--resource-input-file ../pkg/collector/data/json/Example1.json" +
+			" --synthesis-dump-dir examples/output/synthesis --disjoint-hint backend,frontend --disjoint-hint frontend,backend",
```

Comment:
```suggestion
			" --synthesis-dump-dir examples/output/synthesis --disjoint-hint backend,frontend```

Reply:
It is a test to the cli only, so I put it twice. it tests that there is no error getting more than one hint